@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -870,7 +937,25 @@ export class VisibleTileSet {
             }
         });
 
-        this.m_frustumIntersection.updateFrustum();
+        // If elevation is to be taken into account create extended frustum:
+        // (near ~0, far: maxVisibilityRange) that allows to consider tiles that
+        // are far below ground plane and high enough to intersect the frustum.
+        if (elevationRangeSource !== undefined) {
+            const fp = MapViewUtils.getCameraFrustumPlanes(this.m_frustumIntersection.camera);
+            fp.near = this.m_viewRange.minimum;
```

Comment:
This seems to be the only use of the `minimum`, why not just hard code `0.1` and we save a lot of unnecessary code.

Reply:
I think explanation is few comments above, when we discuss the reason for configurable nearMin value.